contributor/making-release.md

```diff
@@ -79,7 +80,8 @@ will make it publicly available for other members to inspect.
 
 At this point, you should have a closed repository that's ready to vote on. Send a message to [the dev
 list](mailto:dev@accumulo.apache.org) and get the ball rolling. Developers should test and verify the
-release candidate on their own. Accumulo has a guide for [verifying releases][verify].
+release candidate on their own. Accumulo has a guide for [verifying releases][verify].  The provided 
+script assemble/build.sh can be used to generate a release VOTE email using the --
```

Comment:
Did this get cut off?

Reply:
It is unfinished.  I marked this as a draft since its still WIP.  I have had it open in my browser with edits as I have done the release but didn't want to lose the changes.